config/clients/python/template/README_installation.mustache

```diff
@@ -0,0 +1,43 @@
+### pip install
+
+#### PyPI
+
+The {{packageName}} is available to be downloaded via PyPI, you can install directly using:
+
+```sh
+pip install {{packageName}}
```

Comment:
Are you sure this works?

For some Linux distros and the current Mac, this has to be: `pip3 install ...`

Reply:
will change to pip3